superchain/chain.go

```diff
@@ -100,6 +100,36 @@ func GetChain(chainID uint64) (*Chain, error) {
 	return BuiltInConfigs.GetChain(chainID)
 }
 
+func GetDepset(chainID uint64) (map[string]Dependency, error) {
+	chain, err := BuiltInConfigs.GetChain(chainID)
+	if err != nil {
+		return nil, err
+	}
+	cfg, err := chain.Config()
+	if err != nil {
+		return nil, err
+	}
+
+	if cfg.Hardforks.InteropTime == nil {
+		return nil, nil
+	}
+
+	// depset of 1 (self) is the default when no dependencies are specified but interop_time is set
+	if cfg.Interop == nil {
+		cfg.Interop = &Interop{
+			Dependencies: make(map[string]Dependency),
+		}
+		self := Dependency{
+			ChainIndex:     1,
```

Comment:
So the chain index is not zero-indexed?

Reply:
It definitely can be if that's preferred. Not sure it matters here but if its preferred for multichain depsets to be zero-indexed, it would make sense to do that here too for consistency

Reply:
I kind of like the idea of avoiding 0 chain ID as a policy. Makes it clear it was actually set.  But that's a very very weak opinion. :)

Reply:
hmm, fun quirk here to deal with - when we "add chains" we're technically merging dependency sets and will need to deal with chain index potentially being different between the original and combined dep set. I don't see a way to avoid that short of using ChainID but it doesn't always fit.

Reply:
> I kind of like the idea of avoiding 0 chain ID as a policy.
This is a chain _index_, not ID. I thought the chain _index_ is just for giving the chains in the dep set a defined order. And for ordering things in a list I find 0-indexing more intuitive.

Reply:
It is really more of an ID than an index. At no point is it used for array lookups, and there is no sorting order other than for display purposes.